docs/REST API/rest-api.md

```diff
@@ -7006,7 +7006,18 @@ Deletes the session specified by ID. When the user associated with the session n
           }
         ],
         "hosts_count": 1
-      }
+      },
+      {
+        "id": 2,
+        "name": "1Password – Password Manager",
+        "version": "2.3.7",
+        "source": "chrome_extensions",
+        "extension_id": "aeblfdkhhhdcdjpifhhbdiojplfjncoa",
+        "browser": "chrome",
+        "generated_cpe": "cpe:2.3:a:1password:1password:2.3.7:*:*:*:*:chrome:*:*",
+        "vulnerabilities": null,
+        "hosts_count": 2
+      },
```

Comment:
@getvictor 

Missing `browser_type` (defined on the corresponding backend PR)

Also, we could document the possible values: https://github.com/osquery/osquery/blob/1028f32e80b33551af6937a30b1b287ea4971820/specs/chrome_extensions.table#L4

Reply:
@lucasmrod It is `browser` for our API. `browser_type` is for the osquery table.